### DISCUSSION TITLE
Ability to show Raw values non temp compensated

### DISCUSSION THREAD

**Discussion by syph3rd** [angrest/pocketBME280#20] [General]
hey angrest, 
any chance to help me modify the library to show uncorrected values as well?

**Comment by angrest** [comment]
Yes, this could be done, but what would be the use case for that?

**Comment by syph3rd** [comment]
I have a device in an enclosure and the PCB temperature is throwing off the readings, I have an external thermistor which I would like to use instead for compensation.

**Comment by angrest** [comment]
The temperature compensation is something which is related to how the measurement works and should be based on the actual temperature at the chip. 
If you want to know the relative humidity corresponding to the temperature measured outside the enclosure, this could be done by calculating the absolute humidity from the measured value and then the relative humidity based on the temperature outside.
